tests/test_2D.py

```diff
@@ -0,0 +1,21 @@
+import subprocess
+import sys
+from pathlib import Path
+
+from .util import get_data
+
+
+def test_output_2D_file_format() -> None:
+    """A simple test for format tag fix for 2D output arrays."""
+
+    Path("filename.txt").touch()
```

Comment:
That creates a file in the current working directory. If you need to create new files, please use the `tmp_path` Pytest fixture

https://github.com/common-workflow-language/cwltool/blob/main/tests/test_examples.py#L1263-L1268